skill-tree/skilltree-database-script.sql

```diff
@@ -0,0 +1,90 @@
+CREATE DATABASE IF NOT EXISTS skilltree;
+USE skilltree;
+
+CREATE TABLE `users` (
+  `id` binary(16) NOT NULL,
+  `created_at` datetime(6) DEFAULT NULL,
+  `updated_at` datetime(6) DEFAULT NULL,
+  `first_name` varchar(255) NOT NULL,
+  `image_url` varchar(255) NOT NULL,
+  `last_name` varchar(255) DEFAULT NULL,
+  `rds_user_id` varchar(255) DEFAULT NULL,
+  `user_role` enum('MAVEN','MEMBER','SUPERUSER','USER') NOT NULL,
+  `created_by` binary(16) DEFAULT NULL,
+  `updated_by` binary(16) DEFAULT NULL,
+  `user_type` tinyint DEFAULT NULL,
+  PRIMARY KEY (`id`),
+  KEY `FKibk1e3kaxy5sfyeekp8hbhnim` (`created_by`),
+  KEY `FKci7xr690rvyv3bnfappbyh8x0` (`updated_by`),
+  CONSTRAINT `FKci7xr690rvyv3bnfappbyh8x0` FOREIGN KEY (`updated_by`) REFERENCES `users` (`id`),
+  CONSTRAINT `FKibk1e3kaxy5sfyeekp8hbhnim` FOREIGN KEY (`created_by`) REFERENCES `users` (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;
+
```

Comment:
I had gone through the design doc for authentication, and in that we are removing the users table 
So will this be included here?

Reply:
For now yes, once we decide will make the changes.